shaka-lab-recommended-settings/windows/README.md

```diff
@@ -0,0 +1,53 @@
+# Shaka Lab Recommended Settings for Windows
+
+The Shaka Lab Recommended Settings package provides recommended OS settings for
+a Shaka Lab environment.
+
+This is the Windows package.
+
+## Pre-requisites
+
+ - Get Chocolatey: [https://docs.chocolatey.org/en-us/choco/setup](https://docs.chocolatey.org/en-us/choco/setup)
+
+## Installation
+
+```sh
+choco source add -n=shaka-lab -s=https://shaka-lab-chocolatey-dot-shaka-player-demo.appspot.com/
+choco install shaka-lab-recommended-settings
+```
+
+## Updates
+
+```sh
+choco upgrade -y shaka-lab-recommended-settings
+```
+
+## Uninstallation
+
+**NOTE**: This package does not back up your original settings.  Uninstalling
+will not restore any settings changed by installation or update of this
+package.
+
+```sh
+choco uninstall -y shaka-lab-recommended-settings
+```
+
+## Specific Settings and Dependencies
+
+Dependencies:
+ - vim (text editor)
+
+Settings:
+ - Install and start SSH server
```

Comment:
This might need some hints like which SSH server to use and where to download it.

Reply:
No, this is not a step the user must take.  It is documentation of steps that are done for you by the package.  In this case, it installs and starts the SSH server provided by Microsoft.  I'll make some updates to the doc to clarify.

Reply:
Got it. Thank you for changing the titles, now it's clear.